UI/index.html

```diff
@@ -1,179 +1,161 @@
+    <meta name="author" content="Niyonsenga Eric" />
+    <title>Way Farer</title>
+    <link href="./css/style.css" rel="stylesheet" type="text/css" />
+    <link href="./css/resp.css" rel="stylesheet" type="text/css" />
   </header>
 
-
-
-
-  <div class="banner">
-    <div id="slides">
-
-    </div>
-    <div class="dark-overlay">
-      <div class="container">
-        <div class="row">
-
-          <h1>Way Farer</h1>
-          <h2>Ticket of Your Desired Destinations</h2>
-          <h3>Cheap and Easy</h3>
-          <p>We Offer You Tickets At Good Price, We also Provide a good Services to our client any time and and any
-            where
-            in our Destinations</p>
-
-          <br>
-          <form>
-
-            <input type="text" id="name" name="name" class="" placeholder="Search Destination">
-
-
-            <button type="submit" href="#" class="button-1">Search</button>
-          </form><br><br>
-
-        </div>
+  <body>
+    <header class="mainheader">
+      <div class="logo">
+        <a href="#"
+          ><h1><b>Way Farer</b></h1></a
+        >
       </div>
-    </div>
-
-  </div>
-
-  <div id="wrapper">
-
-    <div id="recent_newsletter">
-
-      <div class="top">
-        <h2>Latest News</h2>
-        <p>We Provide Good Services to our client at any destination</p>
-      </div>
-      <div id="right">
-
-        <div class="img">
-          <img src="img/trip.jpg">
-
-          <h2><a href="#">Kigali/Huye Public Bus Arive safe</a> </h2>
-          <p>The bus has been arive saferly at the destination</p><br>
-
+      <nav>
+        <div class="menuitems">
+          <ul>
+            <li>
+              <a href="index.html"><b>Home</b></a>
+            </li>
+            <li><a href="bookings.html">Bookings</a></li>
+            <li><a href="book.html">Book Ticket</a></li>
+            <li><a href="dashboard.html">Dashboard</a></li>
+            <li><a href="contact.html">contact Us</a></li>
+          </ul>
         </div>
-
-
-        <div class="img">
-          <img src="img/trip2.jpg">
-
-          <h2><a href="#">Kigali/Huye Public Bus Arive safe</a> </h2>
-          <p>The bus has been arive saferly at the destination</p><br>
-
+      </nav>
+      <form>
+        <input
+          type="text"
+          id="name"
+          name="name"
+          class=""
+          placeholder="Search Destination"
+        />
+
+        <a href="login.html">Login</a>
+        <a href="signup.html">Signup</a>
+      </form>
+    </header>
+
+    <div class="banner">
+      <div id="slides"></div>
+      <div class="dark-overlay">
+        <div class="container">
+          <div class="row">
+            <h2>Ticket of Your Desired Destinations</h2>
+            <h3>Cheap and Easy</h3>
+            <p>
+              We Offer You Tickets At Good Price, We also Provide a good
+              Services to our client any time and and any where in our
+              Destinations
+            </p>
+
+            <br />
+
```

Comment:
Why this space, I think it is not necessary?

Reply:
on the issue of tests am not seeing why it fails but in my local machine it works if there is any help you may help me.

Reply:
ok am going to remove it

Reply:
You need to check your Travis settings for tests but you keep it for now and work on it when you start applying the backend's feedback